test/run-docker.sh

```diff
@@ -34,18 +34,15 @@ if [[ "$(is_running boulder-mysql)" != "true" ]]; then
 	# bring up mysql mariadb container - no need to publish port
 	# 3306 with host networking
 	docker run -d \
-		--net host \
+		-p 3306 \
```

Comment:
As mariadb is only used for internal containers, the port should not be exposed. Otherwise mariadb will be reachable again from the outside world with passwordless setup. Similarly with boulder-rabbitmq.